docs/portal/team-members.md

```diff
@@ -18,17 +18,17 @@ In the UID2 Portal, on the **Team Members** page, you can perform all activities
 
 ## Accessing the Team Members Page
 
-To access the Team Members page, go to the top right of the UID2 Portal user interface where your login ID is displayed. Click the down arrow and choose **Manage Team Members**.
+To access the Team Members page, go to the top right of the UID2 Portal user interface where your name is displayed. Click the down arrow and choose **Manage Team Members**.
 
 For an illustration, see [UID2 Portal Structure](portal-overview.md#uid2-portal-structure).
 
-## Team Member Rights and Responsibilities
+## User Roles
 
-Each team member is an administrator for the account, and can perform all activities relating to managing your team and your account, including the following:
+Each team member has one of two user roles: Admin or Operations. Admins can perform all activities related to UID2 participation, as summarized in the [UID2 Portal Overview](portal-overview.md).
```

Comment:
nit: probably "Admin users" instead of "Admins" makes sense here for consistency

Reply:
yep good call. done.